api/WebSocket.json

```
@@ -1041,10 +1009,10 @@
               "version_added": "10"
             },
             "opera": {
-              "version_added": null
+              "version_added": "≤15"
```

http://mdn-bcd-collector.appspot.com/tests/api/WebSocket?exposure=Worker confirms support in Opera 12.16.

```suggestion
              "version_added": "12.1"
```